src/bootstrap/bin/rustc.rs

```diff
@@ -138,6 +138,10 @@ fn main() {
         cmd.arg("-Z").arg("force-unstable-if-unmarked");
     }
 
+    // This check is currently too expensive to have on by default.
+    // For now, just enable it when building compiler-related crates.
+    cmd.arg("-Z").arg("incremental-verify-ich");
```

Comment:
This would ideally get placed somewhere in https://github.com/rust-lang/rust/blob/master/src/bootstrap/builder.rs#L927, similar to that line (though perhaps not gated by any config).

We're trying to avoid adding any new options to this file and migrate things to rustflags as much as possible.